### PR TITLE
Document getting started.

### PR DESCRIPTION
**Pull request by hodgestar**

### CODE REVIEW

**Comment by hodgestar** [comment]
Just throwing what I have so far up here for while I'm away. Feel free to add to it / land it or what have you, otherwise I'll pick it up again on Monday.

**Comment by hodgestar** [comment]
Script from the demo:
```
sudo docker run -it -p 8123:8123 -p 8124:8124 debian:wheezy /bin/bash
apt-get update
apt-get install redis-server rabbitmq-server python python-dev python-setuptools libffi-dev
easy_install pip virtualenv
virtualenv ve
source ve/bin/activate
pip install junebug
service redis-server start
service rabbitmq-server start
jb -i 0.0.0.0 -p 8123
```

**Comment by justinvdm** [comment]
I think this is ready for review now.

File: docs/getting-started.rst [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+This starts the Junebug HTTP API running on port 8000. At the moment it won't
+have any transports running, so let's create one using the API::
+
+  $ curl -X POST \
+         -d '{
+          "type": "telnet",
+          "label": "My First Channel",
+          "mo_url": "http://requestb.in/pzvivfpz",
+          "config": {
+            "transport_name": "my_first_transport",
```
Comment (rudigiesler) [inline docs/getting-started.rst]: We no longer allow setting the transport name

File: docs/getting-started.rst [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+      }
+
+This message was sent to the channel when we connected to the telnet transport,
+and is equivalent to starting a session for a session-based transport type like USSD.
+
+Now, lets send a message to the telnet transport via junebug::
+
+  $ curl -X POST \
+      -d '{
+        "to": "0.0.0.0:9001",
```
Comment (rudigiesler) [inline docs/getting-started.rst]: This should be the `from` of the received message, ie. `127.0.0.1:53378`

**Comment by rudigiesler** [comment]
Just those two minor comments from me. Looks good otherwise.

**Comment by justinvdm** [comment]
Ready for another review.

**Comment by rudigiesler** [comment]
:+1:

**Pull request merged**